appengine/springboot/src/main/webapp/WEB-INF/appengine-web.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Header

Reply:
I'm not going to put one here because we don't put them in `app.yaml` and I think it'll obscure what is important about this file.